src/select-travel-token-screen/SelectTravelTokenScreenComponent.tsx

```diff
@@ -102,7 +102,7 @@ export const SelectTravelTokenScreenComponent = ({onAfterSave}: Props) => {
         return;
       }
       setSaveState({saving: true, error: false});
-      const success = await toggleToken(selectedToken.id);
+      const success = await toggleToken(selectedToken.id, false);
```

Comment:
This sets `bypassRestrictions` to `false` also when changing to mobile through the `SelectTravelTokenScreen`. This may be what is preferred, but should be "formally" agreed upon so there are no surprises later on how the implementation works 😊

Reply:
Clarified with @Sebstorvik that this is how it should be 👍